test/functional/feature_assumeutxo.py

```diff
@@ -237,6 +237,9 @@ def run_test(self):
         # will allow us to test n1's sync-to-tip on top of a snapshot.
         self.generate(n0, nblocks=100, sync_fun=self.no_op)
 
+        with n0.assert_debug_log(expected_msgs=["[snapshot] activation failed - work does not exceed active chainstate","[snapshot] activation failed - population failed"]):
+            assert_raises_rpc_error(-32603, "Unable to load UTXO snapshot", n0.loadtxoutset, dump_output['path'])
```

Comment:
Also made a temporary modification to this expected message.  Mismatch was caught (as expected).